app/database/migrations/20210213121940_create_clients_table.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('clients', table => {
+        table.integer('job').unsigned().notNullable();
+        table.string('first_name');
+        table.string('last_name');
+        table.string('email', 320);
+
+        table.foreign('job').references('id').inTable('jobs');
```

Comment:
What's this foreign part for?

Reply:
It is to create a foreign key relationship between a client row and a job. Essentially every client row is mapped to a job.